EIPS/eip-draft_grant_standard.md

```diff
@@ -26,69 +26,282 @@ The current process for grants in the Ethereum ecosystem is for membership organ
+```
+function contractExpiration() public view returns(uint256)
+```
+
+#### grantCancelled
+Returns a flag to indicate whether or not grant is cancelled.
+```
+function grantCancelled() public view returns(bool)
+```
+
+#### grantees
+Returns Grantee.
+```
+function grantees(address) public view returns(Grantee)
+```
+
+#### donors
+Returns Donor.
+```
+function donors(address) public view returns(Donor)
+```
+
+#### getAvailableBalance
+Get available grant balance.
+```
+function getAvailableBalance() public view returns(uint256);
+```
+
+#### totalFunding
+Cumulative funding donated by donors.
+```
+function totalFunding() public view returns(uint256);
+```
+
+#### totalPayed
+Cumulative funding payed to grantees.
+```
+function totalPayed() public view returns(uint256);
+```
+
+#### totalRefunded
+OPTIONAL: If refunds permitted, cumulative funding refunded to donors.
+```
+function totalRefunded() public view returns(uint256);
+```
+
+#### pendingPayments
+OPTIONAL: If pull payments used, payments approved to grantees but not yet withdrawn.
+```
+function totalRefunded() public view returns(uint256);
+```
+
+
+
+#### canFund
+Funding status check. true if can fund grant.
+```
+function canFund() public view returns(bool);
+```
+
+### Initialization Methods
+
+#### constructor
+
+Grants are initialized on creation using the constructor. 
+
+```{sol}
+constructor(
+    address[] memory _grantees,
+    uint256[] memory _amounts,
+    address _manager,
+    address _currency,
+    uint256 _targetFunding,
+    uint256 _fundingExpiration,
+    uint256 _contractExpiration
+)
+```
+
+### State Modifying Methods
+
+#### fund
+Fund a grant proposal. `value` in WEI or GRAINS to fund.
+
+NOTE: This method is not `payable`. When funding with Ether, use fallback function to dispatch the `fund` method. 
+```
+function fund(uint256 value) public returns (bool);
+```
+
+#### approvePayout
+OPTIONAL: If managed by a `manager` this method approves payment to a grantee. If using push payouts, will also send payment.
+
+OPTIONAL ARG: Pay to a single `Grantee` when `grantee` address specified otherwise spit payout among `Grantee`s.
+```
+function approvePayout(uint256 value, address grantee) public returns (bool);
+```
+
+#### withdrawPayout
+OPTIONAL: If using pull payments, withdraws portion of the contract's available balance.
+```
+function withdrawPayout(uint256 value, address grantee) public returns (bool);
+```
+
+#### approveRefund
+OPTIONAL: If refunds permitted, approve refunding a portion of the contract's available balance.
+
+OPTIONAL ARG: If `grantee` address specified, reduce `Grantee`'s  `targetFunding`.
+```
+function approveRefund(uint256 value, address grantee) public;
+```
+
+#### withdrawRefund
+Withdraws portion of the contract's available balance to `donor` address.
+```
+function withdrawRefund(address donor) public returns(bool);
+```
+
+#### cancelGrant
+Cancel grant and enable refunds.
```

Comment:
Worth mentioning here that it's an admin-only methods? Do you want to have a modifier to be explicit about this?

Reply:
Expanded the note in an update. Technically anyone can call this function but it will fail unless certain conditions are met (must be grant manager or grant must have missed fundingDeadline, or contractExpiration)